CO2Trade-Login-Register/Service/EntityUsersService.cs

```diff
@@ -104,4 +106,45 @@ public async Task<APIResponse> MyProjects(string idEntityUser)
             return _response;
         }
     }
+
+    public async Task<APIResponse> GetUser(string userId)
+    {
+        try
+        {
+            EntityUser user = await _entityUserRepo.GetUser(userId);
```

Comment:
La clase EntityUserRepository hereda de Repository<EntityUser> el cual contiene el método Task<T> GetAsync(Expression<Func<T, bool>> filter = null, bool tracked = true), que espera una query y devuelve el tipo de Dato de la implementacion, por lo que se podria usar esto para evitar escribir más codigo, de la sig manera:


```suggestion
    EntityUser user = await _entityUserRepo.GetAsync(x => xId==userId);
```

Reply:
Ahi usé el método heredado y funciona joya. El tema es que no puedo hacer Include de rol y entitytype con ese método, entonces de vuelta, no podemos mostrar el rol y el entity type del user. quizas en este caso se justifica porque es para mostrar los datos del perfil de usuario